core/types/printing.cc

```diff
@@ -370,6 +370,8 @@ string AppliedType::show(const GlobalState &gs) const {
     for (auto typeMember : typeMembers) {
         if (typeMember.data(gs)->isFixed()) {
             it = targs.erase(it);
+        } else if (typeMember.data(gs)->name == core::Names::Constants::AttachedClass()) {
+            it = targs.erase(it);
```

Comment:
Huh that's... interesting. I didn't think about how we already had this logic to hide things that are applied types only because they have a fixed type_member and so we'd print them as if they were normal ClassTypes. 👍